BLUESPAWN-client/headers/util/permissions/permissions.h

```diff
@@ -212,6 +260,81 @@ namespace Permissions {
 		 * @return The name of the owner
 		 */
 		virtual std::wstring ToString() const;
+
+		/**
+		* Function to get a list of privileges the owner has
+		*
+		* @return the list of privileges an owner explicity has, and if they 
+		*     are a user, includes the privileges granted to them by the
+		*     groups they're in.
+		*/
+		std::vector<LSA_UNICODE_STRING> GetPrivileges();
```

Comment:
Can this be changed to std::wstrings for easier interoperability with the rest of the code?